external/common/include/fast_cpp_csv_parser/LICENSE

```diff
@@ -0,0 +1,28 @@
+Copyright (c) 2015, ben-strasser
```

Comment:
Just put `csv.h` in `external/common/include/` and leave all the rest of the files out. The `csv.h` file includes the license at the top. And you could add a README.md to the same directory that mentions where `csv.h` came from.